LenovoLegionToolkit.Cmdline/Program.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Reflection;
+using System.Threading.Tasks;
+using LenovoLegionToolkit.Cmdline.Resources;
+using LenovoLegionToolkit.Cmdline.Utils;
+using LenovoLegionToolkit.Lib.Automation;
+using LenovoLegionToolkit.Lib.Automation.CmdLine;
+using LenovoLegionToolkit.Lib.Extensions;
+
+namespace LenovoLegionToolkit.Cmdline;
+
+public class Program
+{
+    public static async Task<int> Main(string[] args)
+    {
+        var flags = new Flags(args);
```

Comment:
I don't know a lot about C# cmd line arg parsing since I spend most of my time writing C++ codes, but as u mentioned I'll take a look, seems to be extension-friendly for the future.